### PR TITLE
Use id instead of host as the identifier of the default instance

### PR DESCRIPTION
**Pull request by allenwq**

### CODE REVIEW

File: app/models/instance.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def find_tenant_by_host(host)
     # @return [Instance]
     def find_tenant_by_host_or_default(host)
       tenants = where do
-        (lower(self.host) == lower(host)) | (self.host == Instance::DEFAULT_HOST_NAME)
+        (lower(self.host) == lower(host)) | (self.id == DEFAULT_INSTANCE_ID)
```
Comment (allenwq) [inline app/models/instance.rb]: @lowjoel This id is needed for squeel right ?

File: app/models/instance.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@ class << self
     #
     # @return [Instance]
     def default
-      @default ||= find_by(host: Instance::DEFAULT_HOST_NAME)
+      @default ||= find_by(id: DEFAULT_INSTANCE_ID)
```
Comment (lowjoel) [inline app/models/instance.rb]: `find(DEFAULT_INSTANCE_ID)`

File: app/models/instance.rb (hunk 1 above)
Comment (lowjoel) [inline app/models/instance.rb]: yep.

File: app/models/instance.rb (hunk 2 above)
Comment (allenwq) [inline app/models/instance.rb]: Here should be find_by because find will raise an exception when record does not exist.

**Comment by lowjoel** [comment]
@allenwq please get @fonglh to fix the seed for both production and staging.

**Pull request merged**